tests/test_fisher.py

```
@@ -226,7 +228,7 @@ def test_matches_scalar_pvalue(self):
         expected_rights = np.array([e[1] for _, e in R_VALIDATED_CASES])
         expected_twos = np.array([e[2] for _, e in R_VALIDATED_CASES])
 
-        lefts, rights, twos = pvalue_npy(a, b, c, d)
+        lefts, rights, twos, odds_ratios = pvalue_npy(a, b, c, d)
```

Ruff (0.16.1)

[warning] Unpacked variable `odds_ratios` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)